### PR TITLE
Add TSL GPU unit test coverage for LTC area lights, point light, SH irradiance, equirect/spritesheet UV, and post-processing utils

### PR DESCRIPTION
**Pull request by bhouston**
This PR adds GPU-executed unit test coverage for another slice of previously-untested TSL functions, using the same GPU test harness as the earlier TSL unit test PRs (`gpuTest()`/`gpuFuzzTest()`, described in [Fine-Grained Shader Validation in Three.js](https://ben3d.ca/blog/fine-grained-shader-validation-in-threejs)).

It also fixes one real bug this pass turned up (see below), and — as a direct consequence of writing the LTC tests — exposes `LTC_Evaluate`, `LTC_Evaluate_Volume`, and `LTC_Uv` from the public `three/tsl` barrel for the first time (previously internal-only, reachable only from `PhysicalLightingModel.js`/`VolumetricLightingModel.js`).

## What's covered

| File | Coverage |
|---|---|
| `TSLBSDFLightingRemainder.tests.js` | `directPointLight()` (general case, no-cutoff fallback, beyond-cutoff edge case, cross-check against `getDistanceAttenuation()`); `getShIrradianceAt()` (9-term spherical harmonics evaluation, several normals, all-zero-coefficients edge case); `LTC_Uv()`, `LTC_Evaluate()`, `LTC_Evaluate_Volume()` (Linearly Transformed Cosines area-light math — general case, behind-the-light-plane edge case, and a dedicated regression test that compiles both LTC functions into the *same* shader, see bug fix below) |
| `TSLUtilsMisc.tests.js` | `equirectUV()`/`equirectDirection()` (formula check, round trip both directions, pole singularity); `spritesheetUV()` (frame→sub-rectangle mapping, with dedicated row-boundary and wraparound off-by-one checks); `interleavedGradientNoise()`, `vogelDiskSample()` (Jimenez 2014 / golden-angle disk sampling); `getScreenPosition()`, `getScreenPositionFromClip()` (clip-space→screen-UV projection) |

## Bug fix: `LTC_Evaluate_Volume()`'s generated shader function was misnamed `LTC_Evaluate`, colliding with the real `LTC_Evaluate()`

**Root cause:** `LTC_Evaluate_Volume`'s `setLayout({ name: ... })` call in `LTC.js` declared `name: 'LTC_Evaluate'` — a copy/paste leftover from `LTC_Evaluate`'s own layout directly above it in the same file. That `name` is exactly what backend codegen emits as the generated shader function's literal name, so both functions would compile down to two *different* function bodies sharing the identical name `LTC_Evaluate`.

**Fix:** changed `LTC_Evaluate_Volume`'s `setLayout({ name: ... })` to `'LTC_Evaluate_Volume'`, matching its own export name.

**Locked down by:** a dedicated regression test in `TSLBSDFLightingRemainder.tests.js` that deliberately calls both `LTC_Evaluate()` and `LTC_Evaluate_Volume()` inside one compiled shader — the exact condition that would trigger the collision — and asserts each still matches its own independently hand-derived formula there.

## Noteworthy TSL behaviors surfaced while writing these tests

Not bugs, and not fixed here — recorded for follow-up:

1. **`equirectUV()`'s U coordinate is genuinely ambiguous at the poles** (`atan2(0,0)`, where every longitude maps to the same point), and — separately — a compile-time-constant pole direction risks the same class of WGSL "indeterminate constant expression" compile hazard already on record for `sinc()`/`getDistanceAttenuation()` in earlier PRs. `v` (the other coordinate) is well-defined and exact at the poles; only `u`'s value is left unasserted in the new test, for this reason.

2. **`getScreenPosition()`/`getScreenPositionFromClip()` divide by `w` with no guard against `w == 0`** — same category as the already-documented `posterize(source, 0)` and `getDistanceAttenuation(cutoffDistance=0)` unguarded-division findings from earlier PRs.

3. **A second, milder naming collision surfaces from the `LTC_Evaluate_Volume` fix above, but is benign**: compiling both LTC functions into the same shader (as the new regression test deliberately does) triggers a console `WARN` — `Declaration name 'LTC_Evaluate' of 'vec3' is a reserved keyword or already in use. Renamed to 'LTC_Evaluate_1'.` — from an auto-generated local variable, not the fixed `setLayout` name. The builder catches and auto-resolves this one itself (correct results, no compile error), but it's worth knowing the `WARN` is expected and explained, not new noise.

### CODE REVIEW

File: build/three.webgpu.js [hunk 1]
```diff
@@ -28968,8 +28968,9 @@ class RotateNode extends TempNode {
 	 * @param {Node} positionNode - The position node.
 	 * @param {Node} rotationNode - Represents the rotation that is applied to the position node. Depending
 	 * on whether the position data are 2D or 3D, the rotation is expressed a single float value or an Euler value.
+	 * @param {string} [order='XYZ'] - The Euler rotation order. Only used for 3D rotation.
 	 */
-	constructor( positionNode, rotationNode ) {
+	constructor( positionNode, rotationNode, order = 'XYZ' ) {
```
Comment (Mugen87) [inline build/three.webgpu.js]: Please remove the build files from the PR.

Comment (bhouston) [reply build/three.webgpu.js]: I've reverted that commit.  It was required in order to have the unit tests pass.  So they will fail now.

Comment (bhouston) [reply build/three.webgpu.js]: Here is the reason why the tests were failing without a build - this addresses it systematically.  I've also partially pulled it into this PR as well to make these CI tests pass: https://github.com/mrdoob/three.js/pull/34362
